### PR TITLE
Onboarding Wizard design loose ends are resolved

### LINKED ISSUE #5068: Onboarding Wizard design loose ends should be resolved
## User Story

As a user, I want the Onboarding Wizard to have consistent design, and without loose ends so that I have a professional and reassuring Onboarding experience.

## Acceptance Criteria
-   [x] Same padidng below steps as appears above steps
-   [x] Focus states should be consistent (Continue button, Step Links)
-   [x] HTML Entities in Country list should be rendered correctly

### PR DESCRIPTION
**Pull request by henryholtgeerts**
Resolves #5068 

## Description
This PR addresses a punchlist of design loose ends associated with the Onboarding Wizard. See the affects and visuals sections below for a complete list, and representation of changes this PR covers.

## Affects
Changes introduced in this PR:
- Consistent focus state for all inputs/elements (SelectInput, CardInput, Buttons, StepLinks, DismissLink)
- Focus state design improved to use dualtone green box-shadow (which ensures that focus state pops on Buttons)
- Focus state design improved to incorporate existing box shadows on Cards in CardInput
- Dismiss Setup Wizard link added to bottom of each step
- CardInput hover state modified to use a material inspired shadow to highligh, rather than setting border color
- HTML Entities in Country list are decoded
- Added padding below steps to match padding above steps
- Button case is consistent with Multi-Step form button case
- Typo in Step 4 is resolved ("PDF Receipts")
- Sets horizontal overflow to hidden for the Wizard, to prevent horizontal scrollbars from appearing
- Removes 'select all that apply' subtitle from cause selection input

## Visuals
Improved focus states, added Dismiss Setup link, and new CardInput hover states (see far right card):
Screen Shot 2020-08-12 at 3 13 39 PM

## Pre-review Checklist
-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Tests included
-   [x] Keyboard accessible
-   [x] Screen reader accessible
-   [ ] Relevant `@since` tags included in DocBlocks
-   [ ] Changelog updated
-   [x] Labels applied if docs are needed
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

## Testing Instructions
Navigate to the Onboarding Wizard (`wp-admin/?page=give-onboarding-wizard`) and proceed through Wizard steps to see/interact with changes implmented in this PR.

### CODE REVIEW

File: assets/src/js/admin/onboarding-wizard/app/steps/addons/index.js [hunk 1]
```diff
@@ -40,7 +41,7 @@ const Addons = () => {
 				</Card>
 				<Card value="pdf-receipts">
 					<PDFReceiptsIcon />
-					<strong>{ __( 'PDF Reciepts', 'give' ) }</strong>
```
Comment (kjohnson) [inline assets/src/js/admin/onboarding-wizard/app/steps/addons/index.js:43]: I never could spell ~Reciepts~ Receipts correctly.

File: assets/src/js/admin/onboarding-wizard/components/card/style.scss [hunk 2]
```diff
@@ -10,6 +10,8 @@
 	flex-direction: column;
 	justify-content: center;
 
+	transition: box-shadow 0.2s ease;
```
Comment (kjohnson) [inline assets/src/js/admin/onboarding-wizard/components/card/style.scss:13]: Nice touch!

File: assets/src/js/admin/onboarding-wizard/components/dismiss-link/style.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.give-obw-dismiss-link {
+	font-family: 'Open Sans', Arial, Helvetica, sans-serif;
+	color: #666;
+	font-weight: 500;
+	font-size: 16px;
+	line-height: 22px;
+	margin-top: 42px;
+	padding: 6px;
```
Comment (kjohnson) [inline assets/src/js/admin/onboarding-wizard/components/dismiss-link/style.scss:8]: I'm not sure why, but the dismiss link padding is forcing a horizontal scroll bar on the Features Step. Same for the Add-ons screen.

With padding:
Screenshot from 2020-08-12 21-01-51

Without padding:
Screenshot from 2020-08-12 21-02-06

File: assets/src/js/admin/onboarding-wizard/components/wizard/style.scss [hunk 4]
```diff
@@ -2,6 +2,7 @@
 	min-height: 100%;
 	height: auto;
 	width: 100%;
+	overflow-x: hidden;
```
Comment (kjohnson) [inline assets/src/js/admin/onboarding-wizard/components/wizard/style.scss:5]: @henryholtgeerts I think this needs to be set at a higher level to resolve the horizontal scroll bar issue.

File: assets/src/js/admin/onboarding-wizard/components/dismiss-link/style.scss (hunk 3 above)
Comment (kjohnson) [reply assets/src/js/admin/onboarding-wizard/components/dismiss-link/style.scss:8]: May be resolved by https://github.com/impress-org/givewp/pull/5077#discussion_r469948424

File: assets/src/js/admin/onboarding-wizard/components/wizard/style.scss (hunk 4 above)
Comment (kjohnson) [reply assets/src/js/admin/onboarding-wizard/components/wizard/style.scss:5]: Looks like it needs to go at the `#onboarding-wizard-app` root.

Comment (henryholtgeerts) [reply assets/src/js/admin/onboarding-wizard/components/wizard/style.scss]: @kjohnson Thanks for pointing out where this should be addressed, was having difficulty replicating the horizontal scrolling issue in browsers locally! Believe that this commit should resolve the overflow issues: https://github.com/impress-org/givewp/commit/4221b69a0c4636ca88a339528dfb11730a267843

**Pull request merged**